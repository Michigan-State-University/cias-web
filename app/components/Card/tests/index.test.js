import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import { Card } from '../index';

describe('<Card />', () => {
  it('should match the snapshot', () => {
    const renderedComponent = renderer.create(<Card />).toJSON();
    expect(renderedComponent).toMatchSnapshot();
  });
});
