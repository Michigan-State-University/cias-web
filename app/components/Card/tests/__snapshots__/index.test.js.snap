// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> should match the snapshot 1`] = `
.c0 {
  width: -webkit-min-content;
  width: -moz-min-content;
  width: min-content;
  height: -webkit-min-content;
  height: -moz-min-content;
  height: min-content;
  padding: 16px;
  background-color: #FFFFFF;
  border-radius: 10px;
}

<div
  className="c0"
/>
`;
